packages/mongo-nav/package.json

```diff
@@ -22,6 +22,7 @@
     "@leafygreen-ui/badge": "^3.0.0",
     "@leafygreen-ui/button": "^4.0.0",
     "@leafygreen-ui/hooks": "^2.0.0",
-    "@leafygreen-ui/icon-button": "^4.0.0"
+    "@leafygreen-ui/icon-button": "^4.0.0",
+    "lodash": "^4.17.15"
```

Comment:
Can we make sure that lodash is a dependency wherever we use it, and then add it to our webpack externals? I think most applications already install it, so there's not much of a need to bundle it.